chia/types/coin_record.py

```diff
@@ -24,3 +26,13 @@ class CoinRecord(Streamable):
     @property
     def name(self) -> bytes32:
         return self.coin.name()
+
+    @property
```

Comment:
I think it's risky to make this a property. It makes it hard to tell that this call has a cost. Since it's a new member function, there shouldn't be any existing code that requires this to be a property.